src/Microsoft.Framework.Runtime/ApplicationEnvironment.cs

```diff
@@ -44,10 +44,12 @@ public string Version
 
         public string Configuration { get; private set; }
 
-
         public FrameworkName RuntimeFramework
         {
-            get { return _targetFramework; }
+            get
+            {
+                return _runtimeEnv.Framework;
+            }
```

Comment:
Would be nice to keep these four lines in one line so as to compact the source files.